pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
     "requests>=0.0.0",
     "ffmpeg-python>=0.2.0",
     "nominal-api==0.708.0",
+    "babel>=2.17.0",
```

Comment:
we already have `dateutil`--is that sufficient for the same use-case? I'd prefer to not add a new dep for a single function call if we can avoid it!